--- a/notas/application/resolvers/meal_resolvers.py	2026-06-09 23:51:09.000000000 +0000
+++ b/notas/application/resolvers/meal_resolvers.py	2026-06-17 17:14:24.091449691 +0000
@@ -238,10 +238,10 @@
 
     "share": {
         "label": "Compartir",
-        "method": "post",
+        "method": "get",
         "icon": "send",
         "order": 90,
-        "desktop_position": "menu",
+        "desktop_position": "inline",
         "mobile_position": "menu",
         "get_url": lambda meal, context=None: reverse(
             "meal_share", args=[meal.id]
@@ -270,8 +270,8 @@
     MEAL_VIEWMODE_PERSONAL_LIST: [
         "list_fork",
         "list_remove",
+        "share",
         "detail",
-        # "share",
     ],
 
     MEAL_VIEWMODE_PERSONAL_DETAIL: [
@@ -280,6 +280,7 @@
         "configure",
         "fork",
         "add_to_dailyplan",
+        "share",
         "remove",
     ],
 
--- a/notas/application/resolvers/dailyplan_resolvers.py	2026-06-09 23:51:09.000000000 +0000
+++ b/notas/application/resolvers/dailyplan_resolvers.py	2026-06-17 17:14:24.092131436 +0000
@@ -160,10 +160,10 @@
 
     "share": {
         "label": "Compartir",
-        "method": "post",
+        "method": "get",
         "icon": "send",
         "order": 90,
-        "desktop_position": "menu",
+        "desktop_position": "inline",
         "mobile_position": "menu",
         "get_url": lambda dp, context=None: reverse(
             "dailyplan_share", args=[dp.id]
@@ -307,8 +307,8 @@
     DAILYPLAN_VIEWMODE_PERSONAL_LIST: [
         "list_fork",
         "list_remove",
+        "share",
         "detail",
-        # "share",
     ],
 
     DAILYPLAN_VIEWMODE_PERSONAL_DETAIL: [
@@ -316,6 +316,7 @@
         "rename",
         "configure",
         "fork",
+        "share",
         "remove",
     ],
 
